script.js

```diff
@@ -0,0 +1,221 @@
+// User starts with either X or O
+// User clicks on a space, and that space shows an X or an O
+  // space displays the X or O, depending on which one was currently in effect
+  // turn switches to other player
+// Play continues until one of two things happens:
+    // No spaces remain
+    // A player, either X or O, has won
+// Reset button at the bottom of the page clears all entries
+
+var squareOne = $("#topRow .left");
+var squareTwo = $("#topRow .middle");
+var squareThree = $("#topRow .right");
+var squareFour = $("#middleRow .left");
+var squareFive = $("#middleRow .middle");
+var squareSix = $("#middleRow .right");
+var squareSeven = $("#bottomRow .left");
+var squareEight = $("#bottomRow .middle");
+var squareNine = $("#bottomRow .right");
+var resetButton = $("button");
+
+var playerTwo = false;
+
+function getWinner () {
```

Comment:
This is a large chunk of code. I think if you revisit it now some ways to refactor will probably come to mind. But for starters, the two portions of this code are pretty much identical save for one item. Could you halve the length of this win condition check by representing that value with an argument?